src/v3/src/OktaSignIn.ts

```diff
@@ -129,28 +176,26 @@ export default class OktaSignIn {
     const { el } = options;
 
     return new Promise<RenderResult>((resolve, reject) => {
-      const onSuccessWrapper = (data: JsonObject): void => {
+      // TODO register "success" hook
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const onSuccessWrapper = (data: RenderResult): void => {
         onSuccess?.(data);
         resolve(data);
       };
+      // TODO register "error" hook
       const onErrorWrapper = (err: Error): void => {
         onError?.(err);
         reject(err);
       };
       try {
-        const target = typeof el === 'string'
-          ? document.querySelector(el)
-          : el;
+        const target = typeof el === 'string' ? document.querySelector(el) : el;
 
         if (target) {
-          render(h(Widget,
-            {
-              events: this.events,
-              authClient: this.authClient,
-              onSuccess: onSuccessWrapper,
-              onError: onErrorWrapper,
```

Comment:
Are we only unregistering these for now because they cause the testcafe tests to fail for some reason?